pycontrails/datalib/ecmwf/era5.py

```diff
@@ -539,9 +539,9 @@ def _preprocess_era5_dataset(self, ds: xr.Dataset) -> xr.Dataset:
             LOG.debug("Input dataset processed with pycontrails > 0.29")
             return ds
 
-        # For "reanalysis-era5-single-levels" or if self.pressure_levels length == 1,
+        # For "reanalysis-era5-single-levels"
         # then the netcdf file does not contain the dimension "level"
-        if len(self.pressure_levels) == 1:
+        if "single-levels" in self.dataset:
```

Comment:
Could you change this to `if self.is_single_level` please?

Reply:
Sure! Great detection, oversaw that param.